lib/dradis/plugins/nexpose/field_processor.rb

```diff
@@ -85,5 +87,20 @@ def format_array_as_table(array)
       end
       rows.join("\n")
     end
+
+    def instantiate_evidence(data)
```

Comment:
idea: if we add the service class `Nexpose::Test`, could it be used here, and also from `Nexpose::Node#tests` and `Nexpose::Service#tests` ?